text/0000-external-doc-attribute.md

```diff
@@ -122,15 +119,26 @@ It should expand out to:
 
 ```rust
 mod example {
-    //! # I'm an example
-    //! This is a markdown file that gets imported to Rust as a doc comment.
+    #![doc("# I'm an example\nThis is a markdown file that gets imported to Rust as a doc comment.")]
     fn my_func() {
       // Hidden implementation
     }
 }
 ```
 
-## Line numbers when errors occur
+## Acceptable Paths
+
+If you've noticed the path given `../docs/example.md` is a relative path to
+`src`. This was decided upon as a good first implementation and further RFCs
+could be written to expand on what syntax is acceptable for paths. For instance
+not being relative to `src`.
+
+## Missing Files or Incorrect Paths
+If a file given to `include` is missing then this should trigger a compilation
+as the given file was supposed to put into the code but for some reason or other
+it is not there.
```

Comment:
I think you accidentally a word there ("...should trigger a compilation [error] as the...")